lesson_06/homework_06.py

```diff
@@ -36,6 +63,15 @@
 Сума цифр числа 12345: 15
 """
 
+sum = 0
+while True:
+    user_input = input("Введіть ціле число: ")
+    if user_input.isdigit():
+        sum += int(user_input)
```

Comment:
```suggestion
        sum += int(user_input)
```
оце неправильно, воно просто повертає те ж число, що ввів користувач